src/app/views/components/date-picker/date-picker.component.ts

```diff
@@ -14,8 +14,8 @@ export class DatePickerComponent {
   public disableDayCallbackDate = new Date();
   public disableDaysOfWeekDate = new Date();
   public disabledDaysOfWeek: number[] = [];
-  public formatCb = (d: Date) => {
-    return d ? d.toISOString().split('T')[0] : null;
```

Comment:
Was getting a type error here because accorded to Forge types, the callback should always return a string, not `null`.  I think this is probably a fine expectation, but we could consider allowing `null` if it's actually handled properly already and might be commonly used.

Reply:
Looks like that would break some assumptions in the date range picker which shares the type.  Might not be too hard to work around, but also not unreasonable to expect people to return `''` instead of `null` given the callback is typed as such.  I think I'd rather leave this for now.